pom.xml

```diff
@@ -93,13 +124,15 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.6</version>
+            <version>${gsonVersion}</version>
             <scope>compile</scope>
         </dependency>
+
+        <!-- Test Dependencies -->
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
-            <version>1.6.0</version>
+            <version>1.14.3</version>
```

Comment:
Can we move this to a property like the others?

Reply:
I just made all the dependencies use properties